tasks/bundle-js.js

```diff
@@ -129,7 +129,7 @@ async function bundleJS(/** @type {JSEntry} */entry, {debug, watch}) {
     });
     entry.watchFiles = bundle.watchFiles;
     await bundle.write({
-        file: `${getDestDir({debug})}/${dest}`,
+        file: `${getDestDir({debug, platform: PLATFORM.CHROME})}/${dest}`,
```

Comment:
Hmm, this will means the watcher function is limited to the chrome folder. We might need to add extra arguments for the watcher then.

Reply:
I think this is intentional, since `bundle.write` creates a generic build (which happens to be Chrome build) and then `entry.postBuild` copies the output to all desired locations simultaniously patching it if needed.

Reply:
Okay.

Reply:
I agree that it is harder to read, but if `bundleJS` was called for every target separatelly, compile time would be almost 3 times longer.